src/Http/Http.Abstractions/src/HttpMethods.cs

```diff
@@ -25,46 +28,109 @@ public static class HttpMethods
         public static readonly string Put = "PUT";
         public static readonly string Trace = "TRACE";
 
+        /// <summary>
+        /// Returns a value that indicates if the HTTP request method is CONNECT. 
+        /// <summary/>
+        /// <param name="method">The HTTP request method.</param>
+        /// <returns>
+        /// The System.Boolean.
+        /// </returns>
         public static bool IsConnect(string method)
         {
             return object.ReferenceEquals(Connect, method) || StringComparer.OrdinalIgnoreCase.Equals(Connect, method);
         }
 
+        /// <summary>
+        /// Returns a value that indicates if the HTTP request method is DELETE. 
+        /// <summary/>
+        /// <param name="method">The HTTP request method.</param>
+        /// <returns>
+        /// The System.Boolean.
+        /// </returns>
         public static bool IsDelete(string method)
         {
             return object.ReferenceEquals(Delete, method) || StringComparer.OrdinalIgnoreCase.Equals(Delete, method);
         }
 
+        /// <summary>
+        /// Returns a value that indicates if the HTTP request method is GET. 
+        /// <summary/>
+        /// <param name="method">The  HTTP request method.</param>
+        /// <returns>
+        /// The System.Boolean.
```

Comment:
Looking at [prior art](https://docs.microsoft.com/en-us/dotnet/api/system.text.rune.isvalid?view=netcore-3.0#System_Text_Rune_IsValid_System_Int32_) this should probably be something like:

```suggestion
        /// <see langword="true" /> if the method is GET; otherwise, <see langword="false" />.
```

(And repeated down the `Is` methods)

Reply:
Or just remove the `<returns>` entirely. The current version seems a little awkward, it's not a `<see>` link and it doesn't tell you what the returned boolean does.